process/block/metablock.go

```diff
@@ -90,6 +91,19 @@ func NewMetaProcessor(arguments ArgMetaProcessor) (*metaProcessor, error) {
 		blockTracker:                 arguments.BlockTracker,
 		dataPool:                     arguments.DataPool,
 	}
+	argsNewEpochStartData := ArgsNewEpochStartData{
```

Comment:
We could extract this block into a function that takes the `arguments` and returns the `epochStartData`

Reply:
done.